### ISSUE TITLE
Can you write an another readme  documation to show how to run this code using my own data?

### ISSUE THREAD

**Issue by ganyibo** [GlassyWing/nvae#1]
Can you write an another readme  documation to show how to run this code using my own data?
I will appreciate it.Thankes

**Comment by GlassyWing** [comment]
> Can you write an another readme documation to show how to run this code using my own data?
> I will appreciate it.Thankes

Hi, you can train the model with your own data with command 
```
python train.py --dataset_path <img_directory> --batch_size 128
```

**Comment by Lukelluke** [comment]
> > Can you write an another readme documation to show how to run this code using my own data?
> > I will appreciate it.Thankes
> 
> Hi, you can train the model with your own data with command
> 
> ```
> python train.py --dataset_path <img_directory> --batch_size 128
> ```

hello, when i wanna change my data to 512*512 size, where should i modify ur implements.
i tried change the '64' into '512' in train.py, however there occur error:
` Sizes of tensors must match except in dimension 1. Got 4 and 32 in dimension 2 at 
/pytorch/aten/src/THC/generic/THCTensorMath.cu:71`

could u please do me a favor

**Comment by GlassyWing** [comment]
The image will be reduced by 32 times, so the initial size should be 512/32 = 16, you need change the map_h and map_w to be 16
https://github.com/GlassyWing/nvae/blob/0fd7dd2e587adea0742ae09dd80f06a7e9a1a655/nvae/decoder.py#L139

**Comment by Lukelluke** [comment]
> The image will be reduced by 32 times, so the initial size should be 512/32 = 16, you need change the map_h and map_w to be 16
> https://github.com/GlassyWing/nvae/blob/0fd7dd2e587adea0742ae09dd80f06a7e9a1a655/nvae/decoder.py#L139

Thank U very much, thank u for teaching me , and I'm now running polished version at Separated Conv2d coding way . As:
`# nn.Conv2d(n_group * dim, n_group * dim, kernel_size=5, padding=2, groups=n_group),
            nn.Conv2d(n_group * dim, n_group * dim, kernel_size=5, stride=1, padding=2, groups=n_group * dim),
            nn.Conv2d(n_group * dim, n_group * dim, kernel_size=1, stride=1, padding=0, groups=1),`
in common.py file
at 1024*1024 images. I will post u the result, hope it could reach original paper performance as much as possible.

Ps. BTW, how do think of why this implement couldn't perform well at 64*64 data? I tried read thorough code and original paper, maybe just in the part of  "BatchNorm" ,there are several tricks mentioned in the paper i couldn't clearly understand how to implement, however this seems not the core secret of NVAE perfect performance.

Beside above mentioned, any part hasn't or couldn't implement in your opinion?

Looking forward to hear from you!

Sincerely,
Luke Huang

**Comment by Lukelluke** [comment]
Well, I noticed that official implement had been released two days ago, good job!
